<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbth_4">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>230</y>
      <width>491</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>choose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>choose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>file 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>file 2(if have)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>file 3(if have)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
